barcode-gen.py

```diff
@@ -36,9 +38,18 @@
 </g>
 </svg>"""
 for (dashed_code, code) in zip(dashed_codes, codes):
-    out_file = code+".svg"
+    print("Generating code:",dashed_code)
+    svg_file = code+'.svg'
+    png_file = code+'.png'
     svg = code128.svg(code)
 
-    with open(out_file,'w') as o:
+    # Generate the svg
+    with open(svg_file,'w') as file:
         svg_content = "\n".join(svg.split("\n")[3:-2])
-        print(template.format(bar=svg_content,code=dashed_code),file=o)
+        print(template.format(bar=svg_content,code=dashed_code),file=file)
+
+    # Convert to png
+    command = ['inkscape', svg_file, '--export-png='+png_file, '--export-width=991', '--export-height=306']
```

Comment:
When this was in the `makefile`, using inkscape seemed reasonable. However if we're doing this all in Python we should look at using something which doesn't need inkscape (it's quite big and not an obvious dependency given that we're using SVG as an _implicit_ intermediary.

There appear to be a number of python libraries which can convert SVG files to PNG; I don't have experience using any of them, though I've used cairo based things elsewhere.